packages/kbn-es/src/cli_commands/archive.js

```diff
@@ -0,0 +1,45 @@
+const dedent = require('dedent');
+const getopts = require('getopts');
+const chalk = require('chalk');
+const { Cluster } = require('../cluster');
+
+exports.description = 'Install and run from an Elasticsearch tar';
+
+exports.usage = 'es archive <path> [<args>]';
+
+const help = dedent`
+  Options:
+
+    --install-path  Directory to install to
```

Comment:
I assume this has a default? Probably worth mentioning.

Reply:
It's the default, but only when run through `scripts/es.js`. We need to know the version and the source directory and I didn't want to reach out from the packages to grab it.

Reply:
Hmm... other tools handle this by having `.rc` files I guess, and can mention those in help bulbs like this...

Reply:
I understand if you don't want to add `.rc` file discovery and parsing to this, what if we passed the defaults from `scripts/es.js` as an argument to `run()` rather than as additional flags in `argv`?

I really hope for `--help` to be a source of truth and not leave people questioning "do I need to pass the `--install-path` flag?"

Reply:
Finally, if we use `process.chdir()` in `scripts/es` to ensure that the current working directory is always the root of the Kibana repo then we can encode defaults into `@kbn/es` that are based of the current working directory.

Reply:
Re-structured by passing defaults to run and it cleaned things up quite a bit: d0d4e71b8075b7cf19a2da559334a349dedc3532

I am not sure that the overhead required in presenting the dynamic defaults in the help test would be worth it.